openedx/features/ondemand_email_preferences/tests/test_helpers.py

```diff
@@ -0,0 +1,82 @@
+"""
+Tests for ondemand email preferences helpers
+"""
+from datetime import datetime
+
+import mock
+from django.conf import settings
+from django.core.urlresolvers import reverse
+
+from openedx.features.ondemand_email_preferences.helpers import (
+    ON_DEMAND_MODULE_TEXT,
+    get_chapters_text,
+    get_my_account_link
+)
+from student.tests.factories import CourseEnrollmentFactory, UserFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
```

Comment:
Please run isort

Reply:
I already did, this is after applying isort.